src/components/SendTokensOne.js

```diff
@@ -156,12 +156,27 @@ class SendTokensOne extends React.Component {
     for (let input of data.inputs) {
       input.tx_id = input.txId;
     }
+    for (let output of data.outputs) {
```

Comment:
This method here was created in the past to speed up hardware wallet integration and is using some old lib methods that will be deprecated soon

I think this is a nice opportunity to refactor this hardware wallet part to use the same methods of the software wallet, instead of continuing adding custom things here just to work specifically for hardware wallet. What do you think?

Reply:
Created a [PR](https://github.com/HathorNetwork/hathor-wallet-lib/pull/306) on the wallet-lib to make the necessary changes for this.